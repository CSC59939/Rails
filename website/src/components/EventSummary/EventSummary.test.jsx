import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Button, Card, Icon } from 'antd';
import { EventSummary } from '..';

configure({ adapter: new Adapter() });

describe("EventSummary", () => {
  it('Render EventSummary without any eror', () => { // eslint-disable-line no-undef
    expect(renderer.create(<EventSummary course="CS103" eventName="Final exam" dueDate="Friday"/>).toJSON()).toMatchSnapshot();
  });
})
