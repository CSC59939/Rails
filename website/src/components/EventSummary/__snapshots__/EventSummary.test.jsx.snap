// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventSummary Render EventSummary without any eror 1`] = `
<div
  className="EventSummary"
  style={
    Object {
      "backgroundColor": "#F44336",
    }
  }
>
  <span
    className="course"
  >
    CSC 59929
  </span>
  <span
    className="eventName"
  >
    Project Presentation
  </span>
  <span
    className="dueDate"
  >
    11:41 PM
  </span>
</div>
`;
